README.md

```diff
@@ -21,7 +22,7 @@ Create a `next.config.js` in your project
 
 ```js
 // next.config.js
-const withVideos = require('next-videos')
+const withVideos = require("next-videos")
```

Comment:
Could you revert all the changes related to single-quotes/double-quotes, semi-colons, bullets points?

Reply:
Sure, no problem, will revert shortly. Thanks for your quick response.

Reply:
Just removed the formatting changes.